common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -6084,6 +6091,18 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  val SSL_HANDSHAKE_TIMEOUT_MS: ConfigEntry[Long] =
+    buildConf("celeborn.ssl.<module>.sslHandshakeTimeoutMs")
+      .categories("network", "ssl")
+      .version("0.6.0")
+      .doc("The timeout for the SSL handshake (in milliseconds). The default value is set to " +
```

Comment:
Which modules could be specified in this config option?

Reply:
We should be use this for `rpc_app` and `rpc_service` transport modules. Added to the description.

Reply:
Where is the added description?

Reply:
My bad, forgot to click the bottom, it's there now.

Reply:
@rmcyang, I mean that you should add the description in this `doc`.

Reply:
oh got it, thanks for clarifying!